### DISCUSSION TITLE
deposit

### DISCUSSION THREAD

**Discussion by coudy999** [Chia-Network/chia-blockchain#18433] [Support]
My funds are sent from OKX via the blockchain but I haven’t received them on this platform. Here are my transaction details: 

Crypto: XCH
Receivable amount: 203.81761137 XCH
Fee: 0.0004 XCH
Network: Chia
Withdrawal platform: OKX
Address: xch12nzsqrjv57mcgtfyn7ckw9lyn6667zcqtdm7kalu56ccuzdysdwssjk6qw
TXID: 0xa4dfcd75e339a24b1504f122383e5b4c4516b17fdbb9e0e87e9591a3eb5b4dea

**Comment by BrandtH22** [comment]
Hey @coudy999 , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

What I recommend trying is a resync of the wallet:

 - If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
 - If using the cli then I would stop chia, rename the wallet db folder in ~.chia\mainnet\wallet\db then restart chia.

If the funds are still not visible can you let us know:

 - if the wallet icon in the top right is orange or green?
 - if orange can you click on the wallet icon and let us know if you see any connected peers?
 - can you provide your log file from ~\.chia\mainnet\log\debug.log

**Comment by coudy999** [comment]
Hello. 
settings -> advanced -> resync,  done.  Funds are still not visible.

Wallet Connections List of connections is empty. 
[debug.log](https://github.com/user-attachments/files/16564276/debug.log)

**Comment by coudy999** [comment]
Everything worked out. Thanks.

**Reply by BrandtH22** [reply]
Thank you for the confirmation, let us know if any other questions arise!
